### ISSUE TITLE
ggplot2 maps on CRAN

### ISSUE THREAD

**Issue by pitkant** [rOpenGov/sorvi#27]
sorvi 0.8.21 was published yesterday (2023-08-22) on CRAN. However, in the [package vignette](https://cloud.r-project.org/web/packages/sorvi/vignettes/sorvi_tutorial.html) there seems to be some problem with drawing maps with ggplot2:

```
library(ggplot2)
map1931 <- get_municipalities(year = 1931)
ggplot(map1931) + geom_sf()
#> Warning in grid.Call.graphics(C_path, x$x, x$y, index, switch(x$rule, winding =
#> 1L, : Path drawing not available for this device
[...]
#> Warning in grid.Call.graphics(C_path, x$x, x$y, index, switch(x$rule, winding =
#> 1L, : Path drawing not available for this device
```

This is not a unique warning, see e.g. https://github.com/r-spatial/sf/issues/1395 and https://github.com/ggseg/ggseg/issues/85. Additionally `Warning in grid.Call.graphics` warnings can be related to something else, such as "semi-transparency is not supported on this device: reported only once per page" in the case of sotkanet README: https://cloud.r-project.org/web/packages/sotkanet/readme/README.html

I wonder if this is a CRAN related issue or if the vignette was generated locally. sessionInfo would seem to suggest that maybe the vignette was generated on your computer @antagomir ?

```
sessionInfo()
#> R version 4.3.0 (2023-04-21)
#> Platform: x86_64-pc-linux-gnu (64-bit)
#> Running under: Ubuntu 22.04.3 LTS
```

**Comment by pitkant** [comment]
I guess that would be one option, but if CRAN maintainers don't react to it then I wonder if it's appropriate to resubmit so early. From [CRAN policies](https://cran.r-project.org/web/packages/policies.html):

> * Once uploaded, no further submissions of that package should be made whilst the uploaded version is pending processing (which may take a few days) and you have not received a reply from a CRAN maintainer.
> * Submitting updates should be done responsibly and with respect for the volunteers’ time. Once a package is established (which may take several rounds), “no more than every 1–2 months” seems appropriate.

Maybe we can just note this and fix it for the next version

**Comment by antagomir** [comment]
Sure but it would nevertheless be good to troubleshoot this now so it is ready for the next. 

Does the html render correctly for you, is this only my local issue?

**Comment by pitkant** [comment]
Yes, it does render correctly for me. Here's a screenshot of my locally rendered version:

<img width="740" alt="sorvi_tutorial" src="https://github.com/rOpenGov/sorvi/assets/69813611/951fb1e7-a6da-4015-8f9d-60b77dd51aca">

**Comment by pitkant** [comment]
With the minor bug fixes I just did the package should be ready for a new CRAN release

**Comment by antagomir** [comment]
Great. 

I still wonder if geofi would be better home for the Finnish geo stuff. @muuankarski should we have a look together?

**Comment by pitkant** [comment]
Undoubtedly. Although I still see these historical mapping functionalities as somewhat experimental / WIP.
